Configurations/shared-info.pl

```diff
@@ -32,6 +32,7 @@ sub detect_gnu_cc {
         return {
             %{$shared_info{'gnu-shared'}},
             shared_defflag    => '-Wl,--version-script=',
+            dso_ldflags => '-z defs',
```

Comment:
Nit: It would be nice if `=>` was aligned with the previous line...

Reply:
Done